kt/godot-runtime/src/main/kotlin/godot/runtime/Registration.kt

```diff
@@ -78,6 +78,32 @@ class ClassBuilderDsl<T : KtObject>(
         )
     }
 
+    fun <P: Any> arrayProperty(
```

Comment:
Not used at the moment. Is this needed? See also my comment on `KtArrayProperty`

Reply:
removed